batch_workflow.py

```diff
@@ -75,14 +86,16 @@
 )
 @click.option(
     '--age-file',
-    'age_data',
+    'age_datas',
```

Comment:
It's nit-picky, but do we want to keep this as `age_data` given that data is a plural noun already?

Reply:
Wanted to be explicit that it can be multiple. But I'll drop those flags soon anyway as we are putting that data under sample-metadata!